apps/ledger-live-mobile/src/screens/Onboarding/steps/newDeviceInfo.tsx

```diff
@@ -95,35 +98,55 @@ function OnboardingStepNewDevice() {
   const route = useRoute<CurrentRouteType>();
   const navigation = useNavigation();
   const { deviceModelId } = route.params;
-
+  const { t } = useTranslation();
   const handleBack = useCallback(() => navigation.goBack(), [navigation]);
 
   return (
     <StyledSafeAreaView>
       <Flex
         flexDirection="row"
-        justifyContent="space-between"
+        justifyContent="center"
         alignItems="center"
         width="100%"
         height={48}
       >
-        <Button Icon={Icons.ArrowLeftMedium} onPress={handleBack} />
-      </Flex>
-      <Carousel onChange={setCurrentIndex}>
-        {new Array(5).fill(null).map((_, index) => (
-          <Card index={index} key={index} deviceModelId={deviceModelId} />
-        ))}
-      </Carousel>
-      <Flex
-        minHeight="60px"
-        width="100%"
-        justifyContent="center"
-        alignItems="center"
-      >
-        <Footer index={currentIndex} />
+        <Button
+          Icon={() => <Icons.ArrowLeftMedium color="neutral.c00" size={24} />}
+          onPress={handleBack}
+          style={styles.backArrow}
+        />
+        <Text variant="h3" mb={3} textAlign="center" color="constant.black">
+          {t(`onboarding.stepNewDevice.${currentIndex}.label`)}
+        </Text>
       </Flex>
+      <InvertTheme>
```

Comment:
I'm not sure what the expected behavior of this, but i think you wanted to use something like this : 
`<ForceTheme selectedPalette={"dark"}>`